scripts/run_pattern_check.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pre-submission script for Oppia.
+# This script checks for unaccepted text patterns in the commit.
+
+import subprocess
+
+BAD_PATTERNS = {'__author__': 'Please remove author tags from \
```

Comment:
Never use backslash. Hard to tell what character is being escaped, e.g. it could be a space at the end of the line, instead of the newline char.

Instead:
```
    BAD_PATTERNS = {
        '__author__': 'Please remove author tags from this file.',
        ...
    }
```

If you need more space:
```
    BAD_PATTERNS = {
        '__author__': (
            'Please remove author tags from this file.'),
        ...
    }
```